src/reverselock.h

```diff
@@ -13,7 +13,7 @@ class reverse_lock
 {
 public:
 
-    explicit reverse_lock(Lock& lock) : lock(lock) {
+    explicit reverse_lock(Lock& _lock) : lock(_lock) {
```

Comment:
Wladimir investigated in #8793. It changes the binaries.

Technically it can be done by changing `lock` to `_lock` in the body of the function. But we decided that it is wrong to do that.